src/components/Filter/Filter.css

```diff
@@ -38,8 +38,15 @@
   }
 }
 
-@media all and (min-width: 577px) {
+@media all and (min-width: 580px) {
```

Comment:
Why this change needed?
Bootstrap using it:

https://getbootstrap.com/docs/4.3/layout/overview/#responsive-breakpoints

Reply:
This one as well 🤔

Reply:
I adjusted the media query widths to match Bootstrap. Originally - I made the changes to make sure that the visual transition when shrinking the screen were as smooth as I could make them.

Reply:
As much as I can see, not all the media queries aligned with bootstrap and not consistently. Why?